services/src/messaging/src/main/java/org/openkilda/messaging/ctrl/state/OFELinkBoltState.java

```diff
@@ -35,7 +35,7 @@ public void accept(DumpStateVisitor visitor) {
         visitor.visit(this);
     }
 
-    public List<?> getDiscovery() {
+    public Set<?> getDiscovery() {
```

Comment:
Would it be possible to return Set<DiscoveryLink> here instead of Set<?>?

Reply:
Fixed.